### DISCUSSION TITLE
How to autocomplete substring of the file.

### DISCUSSION THREAD

**Discussion by irfan-hossain-bhuiyan** [VonHeikemen/lsp-zero.nvim#348] [Q&A]
Let's say I have a text file with a lot of text in it. As usual I have a lot of repeated words.While typing a word,I want neovim will find the substring in the current file,and suggest word by it. As example :
_procastination is a bad habit._ 

After it when I will  type **pro** I would be suggest **procastination** in the autocomplete.
As I know **coc.nvim** do this thing by default. Is there any way to do it in here? It is very useful for editing basic text file,or any string in a programming language.

**Comment by VonHeikemen** [comment]
In the autocomplete documentation there is an example on how to enable that behavior in nvim-cmp: [Autocomplete - adding a source](https://github.com/VonHeikemen/lsp-zero.nvim/blob/v3.x/doc/md/autocomplete.md#adding-a-source)
